layout/_macro/post.swig

```diff
@@ -49,7 +49,7 @@
           {%- if theme.post_meta.created_at %}
             <span class="post-meta-item">
               <span class="post-meta-item-icon">
-                <i class="fa fa-calendar-o"></i>
+                <i class="far fa-calendar"></i>
```

Comment:
`far`?

Reply:
Some `far` are free, and some of the original v4 icons are `far` in v5